appveyor.yml

```diff
@@ -71,3 +74,10 @@ deploy:
     artifact: Pretzel.Logic.nupkg, Pretzel.Logic.symbols.nupkg
     on:
       appveyor_repo_tag: true
+
+  - provider: NuGet
+    api_key:
+      secure: NmM24h0TPsfINvy/1Qr4iJkmRjVztuY5umCJ6mzpxSjvuKl48AXjGEcgow+iJwL8
+    artifact: Pretzel.Tool.nupkg, Pretzel.Tool.symbols.nupkg
```

Comment:
`Pretzel.Tool.symbols.nupkg` haven't been declared in `appveyor.yml` and the content of `Pretzel.Tool.symbols.nupkg` and `Pretzel.Tool.nupkg` looks similar, so I don't know if it is because appveyor upload the same file for the two artifacts or if global tool doesn't handle symbols.

Reply:
Seams really like so. Should I try to include symbols in the tools package and omit the symbols package at all? Or do we need the symbols in practice?

Reply:
I think we can omit the symbols for a global tool.

Reply:
Should be fixed.